docs/packaging/packaging-with-visual-studio.md

```diff
@@ -0,0 +1,84 @@
+## Overview
+This page will cover everything needed to package an application into an isolated Win32 app. 
+
+## Required software pre-requisites:
+* **Visual Studio** version 17.10.2 or higher
+
+## Step 1 - Install the required workloads in Visual Studio including Windows 11 SDK 10.0.26100.0 (or greater)
+Open the Visual Studio Installer and Modify the existing installation. 
+On the Workloads tab check the *Windows application development* box
+(and optionally check the *C++ WinUI app development tools* box for development in C++). 
+The *Windows 11 SDK 10.0.26100.0* (or greater) is also required.
+
+<img src="images/vs/01-VisualStudio-Required-Workloads.png" alt="image" height="600"/>
+
+## Step 2 - Create your app project
+In Visual Studio, create a new C# or C++ project using the template *Blank App, 
+Packaged with Windows Application Packaging Project (WinUI 3 in Desktop)*. 
+Click *Create* and then select *10.0.26100.0* (or later) for Target Platform Version (TPV) 
+and Target Platform Minimum Version (TPMinV). 
+
+<img src="images/vs/02-Blank-App-Packaged-With-Windows-Application-Packaging-Project.png" alt="image" height="150"/>
+
+<img src="images/vs/03-New-Project.png" alt="image" height="500"/>
+
+<img src="images/vs/04-New-Windows-Project.png" alt="image" height="150"/>
+
+## Step 3 - Set NuGet to use Microsoft.Windows.SDK.BuildTools version 10.0.26100.1 or later
```

Comment:
Can we change the title to "Install XXXBuildTools with(on?) NuGet"? Does that make sense? You are not changing `NuGet` right? You are just using it.